src/index.ejs

```diff
@@ -300,21 +300,5 @@
 
   <a name="location"></a>
   <h4 class="header center <%= public._data.baseColor %>-text">Location</h4>
-  <div id="map_wrapper">
-    <div id="map"></div>
-    <div id="map_overlay">
-      <img src="img/vi_logo.jpeg" alt="Virtual Identity AG" width="120px">
-      <h5>Virtual Identity AG</h5>
-      <p class="lead">
-        Isarwinkel 16<br/>
-        81379 München<br/>
-        <a href="http://virtual-identity.com/">http://virtual-identity.com</a>
-      </p>
-      <p class="lead">
-        by U-Bahn: Thalkirchen (U3), 5 minute walk<br/>
-        by Car: There are parking areas nearby
-      </p>
-      <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDWZviFkYEhK8vaV8bXbZ1I0lYRByL0-IA"></script>
-    </div>
+  <h5 class="header center <%= public._data.baseColor %>-text">still looking for a location...</h3>
```

Comment:
This should either be a `h5` or `h3`. Opening tag does not match closing tag.

Reply:
ups - thanks!